root/static/scripts/guess-case/MB/GuessCase/Mode/Base.js

```diff
@@ -1,324 +0,0 @@
-// This file is part of MusicBrainz, the open internet music database.
-// Copyright (C) 2005 Stefan Kestenholz (keschte)
-// Copyright (C) 2015 MetaBrainz Foundation
-// Licensed under the GPL version 2, or (at your option) any later version:
-// http://www.gnu.org/licenses/gpl-2.0.txt
-
-var i18n = require('../../../../common/i18n.js');
-
-var utils = require('../../../utils.js');
-
-MB.GuessCase = (MB.GuessCase) ? MB.GuessCase : {};
-MB.GuessCase.Mode = (MB.GuessCase.Mode) ? MB.GuessCase.Mode : {};
-
-/**
- * Models a GuessCase mode.
- **/
-(function () {
-    var self = {};
-
-    self.fix = function (name, re, replace) {
-        return { name: name, re: re, replace: replace };
-    };
-
-    // ----------------------------------------------------------------------------
-    // member functions
-    // ---------------------------------------------------------------------------
-
-    self.description = "";
-
-    self.getDescription = function () {
-        /* Work around MBS-5734 */
-        return self.description.replace('<a ', '<a target="_blank" ');
```

Comment:
MBS-5734 has been fixed for a while so I removed this from the new file, in favor of a correct solution.